src/man/pam_sss_gss.8.xml

```diff
@@ -0,0 +1,160 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE reference PUBLIC "-//OASIS//DTD DocBook V4.4//EN"
+"http://www.oasis-open.org/docbook/xml/4.4/docbookx.dtd">
+<reference>
+<title>SSSD Manual pages</title>
+<refentry>
+    <xi:include xmlns:xi="http://www.w3.org/2001/XInclude"
+                href="include/upstream.xml" />
+
+    <refmeta>
+        <refentrytitle>pam_sss_gss</refentrytitle>
+        <manvolnum>8</manvolnum>
+    </refmeta>
+
+    <refnamediv id='name'>
+        <refname>pam_sss_gss</refname>
+        <refpurpose>PAM module for SSSD GSSAPI authentication</refpurpose>
+    </refnamediv>
+
+    <refsynopsisdiv id='synopsis'>
+        <cmdsynopsis>
+            <command>pam_sss_gss.so</command>
+            <arg choice='opt'>
+                <replaceable>debug</replaceable>
+            </arg>
+        </cmdsynopsis>
+    </refsynopsisdiv>
+
+    <refsect1 id='description'>
+        <title>DESCRIPTION</title>
+        <para>
+            <command>pam_sss_gss.so</command> authenticates user
+            over GSSAPI in cooperation with SSSD.
+        </para>
+        <para>
+            This module will try to authenticate the user using host@hostname
+            service ticket (which translates to host/hostname@REALM Kerberos
```

Comment:
Hi,

I found this a bit confusing. I would like to suggest to make it a bit more clear what comes from GSSAPI and what is Kerberos by saying something like

    This module will try to authenticate the user using the GSSAPI hostbased service name host@hostname (which translates to host/hostname@REALM Kerberos principal).

Additionally I would suggest to mention the the `REALM` is derived with the help of the `[domain_realm]` section in `/etc/krb5.conf` which might become important if the fully-qualified name of the host is from a DNS domain which does not match the realm.

bye,
Sumit

Reply:
See the changes. I'm not very sure about the wording.